doc/settings/smtp.md

```diff
@@ -77,6 +77,18 @@ gitlab_rails['smtp_password'] = "8b6ffrmle180"
 gitlab_rails['smtp_domain'] = "mg.gitlab.com"
 ```
 
+### QQ Mail
+
+```ruby
+gitlab_rails['smtp_enable'] = true
+gitlab_rails['smtp_address'] = "smtp.qq.com"
+gitlab_rails['smtp_port'] = 25
```

Comment:
@axilleas QQ mail uses port 25 as its default port, I have tested it on my computer.